hack/run-dockershim-critest.sh

```diff
@@ -16,6 +16,9 @@
 
 # Run critest with default dockershim.
 
+# Works only with https://github.com/Mirantis/cri-dockerd - other CRI
```

Comment:
This information is incorrect, it still sets the endpoint to dockershim (not cri-dockerd) below.